plugins/modules/aws_glue_crawler.py

```diff
@@ -0,0 +1,409 @@
+    returned: when state is present
+    type: complex
+    contains:
+        RecrawlBehavior:
+            description: Whether to crawl the entire dataset again or to crawl only folders that were added since the last crawler run.
+            returned: when state is present
+            type: str
+            sample: CRAWL_EVERYTHING
+role:
+    description: The name or ARN of the IAM role associated with this crawler.
+    returned: when state is present
+    type: str
+    sample: my-iam-role
+schema_change_policy:
+    description: The policy for the crawler's update and deletion behavior.
+    returned: when state is present
+    type: complex
+    contains:
+        DeleteBehavior:
+            description: The deletion behavior when the crawler finds a deleted object.
+            returned: when state is present
+            type: str
+            sample: DELETE_FROM_DATABASE
+        UpdateBehavior:
+            description: The update behavior when the crawler finds a changed schema.
+            returned: when state is present
+            type: str
+            sample: UPDATE_IN_DATABASE
+
+table_prefix:
+    description: The table prefix used for catalog tables that are created.
+    returned: when state is present
+    type: str
+    sample: my_prefix
+targets:
+    description: A list of targets to crawl.
+    returned: when state is present
+    type: complex
+    contains:
+        S3Targets:
+            description: List of S3 targets.
+            returned: when state is present
+            type: list
+        JdbcTargets:
+            description: List of JDBC targets.
+            returned: when state is present
+            type: list
+        MongoDBTargets:
+            description: List of Mongo DB targets.
+            returned: when state is present
+            type: list
+        DynamoDBTargets:
+            description: List of DynamoDB targets.
+            returned: when state is present
+            type: list
+        CatalogTargets:
+            description: List of catalog targets.
+            returned: when state is present
+            type: list
+'''
+
+# Non-ansible imports
+import copy
+try:
+    import botocore
+except ImportError:
+    pass  # Handled by AnsibleAWSModule
+
+from ansible.module_utils.common.dict_transformations import camel_dict_to_snake_dict
+
+from ansible_collections.amazon.aws.plugins.module_utils.core import AnsibleAWSModule
+from ansible_collections.amazon.aws.plugins.module_utils.core import is_boto3_error_code
+from ansible_collections.amazon.aws.plugins.module_utils.ec2 import ansible_dict_to_boto3_tag_list
+from ansible_collections.amazon.aws.plugins.module_utils.ec2 import AWSRetry
+from ansible_collections.amazon.aws.plugins.module_utils.ec2 import boto3_tag_list_to_ansible_dict
+from ansible_collections.amazon.aws.plugins.module_utils.ec2 import compare_aws_tags
+
+
+def _get_account_info(module):
```

Comment:
We have a helper in amazon.aws.iam.get_aws_account_info you can use here

Reply:
Thanks, I replaced the custom method with the helper.